CONTRIBUTING.md

```diff
@@ -169,40 +177,53 @@ packages/babel-plugin-transform-arrow-functions/test/fixtures/arrow-functions/de
 You can use:
 
 ```sh
-$ TEST_GREP="arrow functions destructuring parameters" make test
+$ yarn jest -t "arrow functions destructuring parameters"
 ```
 
-Or you can directly use Yarn:
+Or you can use the `TEST_GREP` environment variable:
 
 ```sh
-$ yarn jest -t "arrow functions destructuring parameters"
+$ TEST_GREP="arrow functions destructuring parameters" make test-only
 ```
 
 #### Run test with Node debugger
 
-To enable the Node.js debugger, set the <code>TEST_DEBUG</code> environment variable:
+To enable the Node.js debugger, run node with `--inspect-brk` option and include the [Jest CLI `-i` option](https://jestjs.io/docs/cli#--runinband) or you [may not hit breakpoints with the chrome debugger](https://github.com/nodejs/node/issues/26609).
 
 ```sh
-$ TEST_DEBUG=true make test
+yarn run --inspect-brk jest -i packages/package-to-test
 ```
 
 <details>
   <summary>More options</summary>
 
-  You can also run `jest` directly, but you must remember to include `--runInBand` or `-i` or you [may not hit breakpoints with the chrome debugger](https://github.com/nodejs/node/issues/26609).
+You can also set the <code>TEST_DEBUG</code> environment variable
+
+```sh
+$ TEST_DEBUG=true make test-only
+```
 
-  ```sh
-  yarn run --inspect-brk jest -i packages/package-to-test
-  ```
 </details>
 <br>
 
-You can combine `TEST_DEBUG` with `TEST_GREP` or `TEST_ONLY` to debug a subset of tests. If you plan to stay long in the debugger (which you'll likely do!), you may increase the test timeout by editing [test/testSetupFile.js](https://github.com/babel/babel/blob/main/test/testSetupFile.js).
+You can combine `-i` with `-t` to debug a subset of tests. For example,
+
+```sh
+yarn run --inspect-brk jest -i babel-plugin-arrow-functions -t "destructuring parameters"
+```
+
+If you plan to stay long in the debugger (which you'll likely do!), you may increase the test timeout by editing [test/testSetupFile.js](https://github.com/babel/babel/blob/main/test/testSetupFile.js).
```

Comment:
This file has been removed. To increase the timeout we can set the `--testTimeout` directly.